### PR TITLE
Fix "make test"

### PR DESCRIPTION
**Pull request by adrianmoisey**
It previously failed with this error:
```
$ make test
Fetching https://github.com/hashicorp/copywrite/releases/download/v0.16.3/SHA256SUMS
Fetching https://github.com/hashicorp/copywrite/releases/download/v0.16.3/copywrite_0.16.3_darwin_x86_64.tar.gz
copywrite_0.16.3_darwin_x86_64.tar.gz: OK
mv: rename /var/folders/kv/rdx7k58x5fj0wb03vgdh6zqh0000gp/T/tmp.wU0r4D42/copywrite to bin/copywrite-0.16.3: No such file or directory
make: *** [copywrite] Error 1
```

It seems like the destination "bin" direcory doesn't exist and needs creating.

### CODE REVIEW

File: hack/install_copywrite.sh [hunk 1]
```diff
@@ -41,6 +41,7 @@ pushd $tempdir > /dev/null
 popd > /dev/null
 
 rm -f "${dest_file%/*}"/copywrite*
+mkdir bin/
```
Comment (kschoche) [inline hack/install_copywrite.sh:44]: I think this would fail the second time it was run without `-p`?

```suggestion
mkdir -p bin/
```

Comment (adrianmoisey) [reply hack/install_copywrite.sh]: Yup, my bad! I've applied your suggestion.

**Review by kschoche — APPROVED** [review]
Looks great, thank you!

**Comment by adrianmoisey** [comment]
\o/

I can't merge It yet, as the tests haven't run.
I'm unsure how to get them started.

**Comment by kschoche** [comment]
> \o/
> 
> I can't merge It yet, as the tests haven't run. I'm unsure how to get them started.

Sorry this is a problem on our end with our CI, it's not running against forks for some reason, working on it, but will merge for you in the meantime!

**Comment by kschoche** [comment]
> > \o/
> > I can't merge It yet, as the tests haven't run. I'm unsure how to get them started.
> 
> Sorry this is a problem on our end with our CI, it's not running against forks for some reason, working on it, but will merge for you in the meantime!

Actually it looks like I don't have permissions to do that even, I'll try to circle back to this. Thanks again!

**Comment by adrianmoisey** [comment]
No worries, and no rush. This is a minor tweak, nothing serious.

**Review by tvoran — APPROVED** [review]

**Pull request merged**